bin/morgue.js

```diff
@@ -6787,10 +6787,8 @@ function coronerDelete(argv, config) {
   }
 
   if (aq && aq.query) {
-    coroner.promise('query', p.universe, p.project, aq.query).then(function(r) {
-      unpackQueryObjects(o, r);
-      return delete_fn();
-    }).then(std_success_cb).catch(std_failure_cb);
+    coroner.promise('delete_by_query', p.universe, p.project, aq.query, params)
+      .then(std_success_cb).catch(std_failure_cb);
```

Comment:
We might want to leave the existing client-side execution path as an option, so it can still be executed on servers that don't have delete-by-query yet.